api/src/main/resources/leonardo.yaml

```diff
@@ -1896,11 +1896,16 @@ components:
     AppStatus:
       type: string
       enum:
-        - DELETING
         - DELETED
+        - DELETING
```

Comment:
These values are present in Leonardo code and returned by endpoints.  They're just missing in the Swagger.  PR to update the Swagger at the source: https://github.com/DataBiosphere/leonardo/pull/3121

Reply:
Leo Swagger updated.  The main thing here is the addition of the STARTING status.